MachineLearning_Engine/Compute/LinearRegression.py

```diff
@@ -25,8 +25,8 @@
 
 
 def fit(x: np.ndarray, y: np.ndarray):
-	assert x.shape == y.shape, "Input data and target data have different shape {} and {}".format(
-								x.shape, y.shape)
+	assert x.shape[0] == y.shape[0], "Input data and target data have different shape {} and {}".format(
```

Comment:
Uhm, not sure about this change. What was the issue with checking the entire shape? This way it only controls the number of samples, correct?

Reply:
The original one will fail for 2 dimension tensors. When I plug 2 dimension tensor in, it is giving error message that x has shape (50,2) while y has shape (50,).

Reply:
Oh, sorry this is my bad, agreed! 
Maybe then `len(x) == len(y)` is a better choice?